dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/entity/Command.java

```diff
@@ -61,6 +61,7 @@ public class Command {
     private String commandParam;
 
     @TableField("task_depend_type")
+    @Builder.Default
```

Comment:
Please add `@Builer.Default` to `failureStrategy` and `updateTime`.

In fact I don't think it's a good idea to add default value in DO， but this is a history issue.

Reply:
Added